### ISSUE TITLE
Pyomo times out before CBC

### ISSUE THREAD

**Issue by victorgarcia98** [Pyomo/pyomo#2029]
Hello,

## Summary

I'm trying to deploy a model using flask in AWS (inside a docker container) and I'm facing an error. I'm using the solver CBC to solve MILP. In some instances of the problem the script fails with the message found below:

```console
Command '['/srv/flask_app/~/cbc/bin/cbc', '-sec', '100', '-timeMode', 'elapsed', '-printingOptions', 'all', '-import', '/tmp/tmp_ex0d7xd.pyomo.lp', '-stat=1', '-solve', '-solu', '/tmp/tmp_ex0d7xd.pyomo.soln']' timed out after 101 seconds
bg.py finished
```

The weird thing is that the same script with the same inputs runs without complaining locally (Mac M1). I suspect that cbc is hanging or taking longer than 100s but the python is expecting it to take <=100s, thus it times out. 

### Information on your system

Pyomo version: Pyomo 5.7.3
Python version: Python 3.8.5
Operating system: Ubuntu
How Pyomo was installed (PyPI, conda, source): conda
Solver (if applicable): Cbc

Please let me know any extra detail you need.

Thanks a lot for your help,
Víctor

**Comment by blnicho** [comment]
A few recommendations and questions:

1. Are you setting the `timelimit` option when calling the `solve` method? If so, try removing this option and look at the solver log on AWS to see what's going on in the solver.

2. Try updating your Pyomo version to 6.0.1. There were a few fixes to the CBC interface merged between the 5.7.3 and 6.0 releases.

3. What version of CBC is being used in AWS and is it the same as the version on your local machine?

**Comment by blnicho** [comment]
Closing this as no additional information has been provided. @victorgarcia98 feel free to reopen if this is still an issue for you.
